cobalt/web/environment_settings.h

```diff
@@ -46,13 +48,16 @@ class EnvironmentSettings : public script::EnvironmentSettings {
   // https://storage.spec.whatwg.org/#obtain-a-storage-key
   url::Origin ObtainStorageKey() { return url::Origin::Create(creation_url()); }
 
+  bool cors_disabled() { return cors_disabled_; }
+
  protected:
   friend std::unique_ptr<EnvironmentSettings>::deleter_type;
 
  private:
   DISALLOW_COPY_AND_ASSIGN(EnvironmentSettings);
 
   Context* context_ = nullptr;
+  bool cors_disabled_ = false;
```

Comment:
Can you make it where this member variable doesn't even exist in gold builds? See similar implementation of kCspEnforcementDisable with the COBALT_FORCE_CSP define.